frontend/src/actions/receive/draw_request_action.ts

```diff
@@ -0,0 +1,16 @@
+import type { Game } from '@/models/game'
+
+class DrawRequestAction {
```

Comment:
Actions must be verbs, actions (`MovePiece`, `Abandon`, etc.). `DrawRequest ` is a noun, it's not intuitive what it does. `RequestDraw`, `ReceiveDraw`, `AnswerDraw`, `ReceiveDrawAnswer` are examples of proper action names. If I understood correctly this implementation, this actions is receiving the answer of a draw request, so a possible name might be `ReceiveDrawAnswer`.